### ISSUE TITLE
Advanced installs can't configure VPN always-on functionality

### ISSUE THREAD

**Issue by grempe** [trailofbits/algo#199]
The code provided in `deploy.yml` and described in the ADVANCED.md file:

https://github.com/trailofbits/algo/blob/master/docs/ADVANCED.md#scripted-deployment

does not seem to provide an equivalent to the interactive `algo` script to allow specifying that you want the VPN to be configured always on when on a cellular or wifi network.

**Comment by grempe** [comment]
See PR https://github.com/trailofbits/algo/pull/203

**Comment by grempe** [comment]
Since my PR #203 was closed without merge, I wanted to document how I proposed informing people to be able to accomplish this. After figuring this out, I'm not sure if this is actually a bug, or a feature request currently lacking a more polished way to handle it.

I believe one way to accomplish this is by adding one, or both, of the following to the `-e` extra variables option for `ansible-playbook`. Not sure if this is the solution you want to end up with but it seems to work in my testing.

```
OnDemandEnabled_WIFI=Y
OnDemandEnabled_Cellular=Y
```
